src/App.css

```diff
@@ -1,33 +1,169 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 .App {
   text-align: center;
 }
 
+.app-title {
+  font-family: 'Montserrat', sans-serif;
+}
+
 nav {
   display: flex;
+  background-color: white;
+  padding-top: 2rem;
+  padding-bottom: 2rem;
   justify-content: space-between;
-  width: 87.5rem;
-  height: 5.938rem;
-  margin: 0 0 2.375rem;
-  background-color: var(--white-two);
+  align-items: center;
 }
 
-.title-links {
-  display: flex;
+a {
+  text-decoration: none;
 }
 
-.nav-links {
+ul {
   display: flex;
+  gap: 0.5rem;
   list-style-type: none;
-  gap: 1rem;
 }
 
-.nav-links li a {
+ul li {
   text-decoration: none;
 }
 
+ul li a {
+  color: gray;
+}
+
+ul li a:hover {
+  color: #00a8ff;
+}
+
+ul li a.active {
+  color: black;
+}
+
+.title-links {
+  display: flex;
+  align-items: center;
+  gap: 2rem;
+}
+
+.title-links h1 {
+  color: #0290ff;
+}
+
+img {
+  width: 30px;
+}
+
 .book-item {
   display: flex;
+  background-color: white;
+  padding: 1rem 2rem 1rem 2rem;
   margin: 0 0 2.375rem;
   border: 1px solid lightgrey;
   justify-content: space-between;
 }
+
+.book-info {
+  display: flex;
+  flex-direction: column;
+}
+
+.book-title {
+  font-family: 'RobotoSlab-Bold', sans-serif;
+  font-weight: 700;
+  font-size: 1.375rem;
+  color: #121212;
+}
+
+.category {
+  font-family: 'Montserrat-Bold', sans-serif;
+  color: gray;
+  border-radius: 4px;
+  width: 30%;
+  font-size: 0.875rem;
+}
+
+.actions {
+  display: flex;
+  color: #4386bf;
+  margin-top: 1rem;
+  gap: 0.5rem;
+}
+
+.author {
+  color: lightblue;
+}
+
+.remove-btn {
+  background: none;
+  color: inherit;
+  border: none;
+  padding: 0;
+  font: inherit;
+  cursor: pointer;
+  outline: inherit;
+}
+
+.progress {
+  display: flex;
+  gap: 0.5rem;
+}
+
+.loadingprogress {
+  width: 50px;
+  height: 50px;
+}
+
+.update-progress-btn {
+  background-color: #008cba;
+  border: none;
+  border-radius: 5px;
+  color: white;
+  padding: 15px 32px;
```

Comment:
- Your button is too big compared to the original design. 8px top and bottom padding would make it better. please, fix this issue.